server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java

```diff
@@ -0,0 +1,202 @@
+      "insert into root.ln.wf02.wt02(timestamp,status) values(1509466140000,false)",
+      "set storage group to root.sgcc",
+      "create timeseries root.sgcc.wf03.wt01.status with datatype=BOOLEAN,encoding=PLAIN",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465600000,true)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465660000,true)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465720000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465780000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465840000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465900000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509465960000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466020000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466080000,false)",
+      "insert into root.sgcc.wf03.wt01(timestamp,status) values(1509466140000,false)",
+      "create timeseries root.sgcc.wf03.wt01.temperature with datatype=FLOAT,encoding=RLE",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465600000,25.957603)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465660000,24.359503)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465720000,20.092794)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465780000,20.182663)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465840000,21.125198)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465900000,22.720892)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509465960000,20.71)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466020000,21.451046)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466080000,22.57987)",
+      "insert into root.sgcc.wf03.wt01(timestamp,temperature) values(1509466140000,20.98177)",
+  };
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    EnvironmentUtils.closeStatMonitor();
+    daemon = IoTDB.getInstance();
+    daemon.active();
+    EnvironmentUtils.envSetUp();
+
+    importData();
+  }
+
+  @AfterClass
+  public static void tearDown() throws Exception {
+    daemon.stop();
+    EnvironmentUtils.cleanEnv();
+  }
+
+  private static void importData() throws ClassNotFoundException, SQLException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+
+      for (String sql : sqls) {
+        statement.execute(sql);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
You should fail it if there does be an exception.

Reply:
fixed